core/azure-core/src/main/java/com/azure/core/implementation/logging/ServiceLogger.java

```diff
@@ -35,39 +39,30 @@ public ServiceLogger(String className) {
     }
 
     /**
-     * Sets the logger to the trace logging level.
-     * @return Updated ServiceLogger if trace is enabled, otherwise a no-op logger.
-     */
-    @Override
-    public ServiceLoggerAPI asTrace() {
-        return asLevel(TRACE_LEVEL);
-    }
-
-    /**
-     * Sets the logger to the debug logging level.
+     * Sets the logger to the verbose logging level.
      * @return Updated ServiceLogger if debug is enabled, otherwise a no-op logger.
      */
     @Override
-    public ServiceLoggerAPI asDebug() {
-        return asLevel(DEBUG_LEVEL);
+    public ServiceLoggerAPI asVerbose() {
+        return asLevel(VERBOSE_LEVEL);
     }
 
     /**
-     * Sets the logger to the info logging level.
+     * Sets the logger to the infomational logging level.
      * @return Updated ServiceLogger if info is enabled, otherwise a no-op logger.
      */
     @Override
     public ServiceLoggerAPI asInformational() {
```

Comment:
I've always slightly disliked this name - what do you think of shortening it to `asInfo`?